Condition.ts

```diff
@@ -49,9 +51,11 @@ export default class Condition {
 
     private static parseRegExp(pattern: string, condition_str: string, matching_address: boolean): RegExp {
         assert(pattern.length > 0, `Condition ${condition_str} should have value but not found`);
-        if (pattern.startsWith('/') && pattern.endsWith('/')) {
+        if (pattern.startsWith('/') && RE_FLAG_PATTERN.test(pattern)) {
             // normal regexp
-            return new RegExp(pattern.substring(1, pattern.length - 1));
+            const p = pattern.substring(1).replace(RE_FLAG_PATTERN, '');
+            const flags = pattern.match(RE_FLAG_PATTERN)![0]!.substring(1);
+            return new RegExp(p, flags);
```

Comment:
How about using regexp `^/(.*)/([gimuy]*)$` to extract both parts at the same time?